<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HireArmy</class>
 <widget class="QDialog" name="HireArmy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="soldier_btn_1">
           <property name="text">
            <string>Kunnka</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_2">
           <property name="text">
            <string>Rexxar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_3">
           <property name="text">
            <string>Bradwarden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_4">
           <property name="text">
            <string>Raigor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_5">
           <property name="text">
            <string>Purist</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_6">
           <property name="text">
            <string>Mangix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_7">
           <property name="text">
            <string>Sven</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_8">
           <property name="text">
            <string>Tiny</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_9">
           <property name="text">
            <string>Cairne</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_10">
           <property name="text">
            <string>Rooftrellen</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_11">
           <property name="text">
            <string>Razzil</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_12">
           <property name="text">
            <string>Rigwarl</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_13">
           <property name="text">
            <string>Rattletrap</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_14">
           <property name="text">
            <string>Magnus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_btn_15">
           <property name="text">
            <string>Huskar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_1">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_2">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_3">
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_4">
           <property name="text">
            <string>25</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_5">
           <property name="text">
            <string>50</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_6">
           <property name="text">
            <string>75</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_7">
           <property name="text">
            <string>100</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_8">
           <property name="text">
            <string>250</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_9">
           <property name="text">
            <string>500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_10">
           <property name="text">
            <string>750</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_11">
           <property name="text">
            <string>1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_12">
           <property name="text">
            <string>2500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_13">
           <property name="text">
            <string>5000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_14">
           <property name="text">
            <string>7500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_dmg_lbl_15">
           <property name="text">
            <string>10000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="soldier_price_lbl_1">
           <property name="text">
            <string>1</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_2">
           <property name="text">
            <string>5</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_3">
           <property name="text">
            <string>10</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_4">
           <property name="text">
            <string>25</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_5">
           <property name="text">
            <string>50</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_6">
           <property name="text">
            <string>75</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_7">
           <property name="text">
            <string>100</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_8">
           <property name="text">
            <string>250</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_9">
           <property name="text">
            <string>500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_10">
           <property name="text">
            <string>750</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_11">
           <property name="text">
            <string>1000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_12">
           <property name="text">
            <string>2500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_13">
           <property name="text">
            <string>5000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_14">
           <property name="text">
            <string>7500</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_price_lbl_15">
           <property name="text">
            <string>10000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="soldier_2_btn_1">
           <property name="text">
            <string>Crixalis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_2">
           <property name="text">
            <string>Barathrum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_3">
           <property name="text">
            <string>Leviathan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_4">
           <property name="text">
            <string>Nessaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_5">
           <property name="text">
            <string>Lucifer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_6">
           <property name="text">
            <string>Abaddon</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_7">
           <property name="text">
            <string>Azgalor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_8">
           <property name="text">
            <string>Slardar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_9">
           <property name="text">
            <string>Dirge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_10">
           <property name="text">
            <string>Magina</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_11">
           <property name="text">
            <string>Yurnero</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_12">
           <property name="text">
            <string>Slithice</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_13">
           <property name="text">
            <string>Slithice</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_14">
           <property name="text">
            <string>Traxex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="soldier_2_btn_15">
           <property name="text">
            <string>Darkterror</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_1">
           <property name="text">
            <string>25000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_2">
           <property name="text">
            <string>50000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_3">
           <property name="text">
            <string>75000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_4">
           <property name="text">
            <string>100000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_5">
           <property name="text">
            <string>125000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_6">
           <property name="text">
            <string>150000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_7">
           <property name="text">
            <string>175000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_8">
           <property name="text">
            <string>200000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_9">
           <property name="text">
            <string>500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_10">
           <property name="text">
            <string>1000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_11">
           <property name="text">
            <string>2500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_12">
           <property name="text">
            <string>5000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_13">
           <property name="text">
            <string>7500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_14">
           <property name="text">
            <string>10000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_dmg_lbl_15">
           <property name="text">
            <string>50000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_1">
           <property name="text">
            <string>25000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_2">
           <property name="text">
            <string>50000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_3">
           <property name="text">
            <string>75000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_4">
           <property name="text">
            <string>100000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_5">
           <property name="text">
            <string>125000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_6">
           <property name="text">
            <string>150000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_7">
           <property name="text">
            <string>175000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_8">
           <property name="text">
            <string>200000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_9">
           <property name="text">
            <string>500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_10">
           <property name="text">
            <string>1000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_11">
           <property name="text">
            <string>2500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_12">
           <property name="text">
            <string>5000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_13">
           <property name="text">
            <string>7500000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_14">
           <property name="text">
            <string>10000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="soldier_2_price_lbl_15">
           <property name="text">
            <string>50000000</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
